chia/plotting/manager.py

```diff
@@ -254,154 +253,145 @@ def refresh_batch(self, plot_paths: List[Path], plot_directories: Set[Path]) ->
         if self.match_str is not None:
             log.info(f'Only loading plots that contain "{self.match_str}" in the file or directory name')
 
-        def process_file(file_path: Path) -> Dict:
-            new_provers: Dict[Path, PlotInfo] = {}
+        def process_file(file_path: Path) -> Optional[PlotInfo]:
             filename_str = str(file_path)
             if self.match_str is not None and self.match_str not in filename_str:
-                return new_provers
-            if file_path.exists():
-                if (
-                    file_path in self.failed_to_open_filenames
-                    and (time.time() - self.failed_to_open_filenames[file_path])
-                    < self.refresh_parameter.retry_invalid_seconds
-                ):
-                    # Try once every `refresh_parameter.retry_invalid_seconds` seconds to open the file
-                    return new_provers
-                if file_path in self.plots:
-                    try:
-                        stat_info = file_path.stat()
-                    except Exception as e:
-                        log.error(f"Failed to open file {file_path}. {e}")
-                        return new_provers
-                    if stat_info.st_mtime == self.plots[file_path].time_modified:
-                        new_provers[file_path] = self.plots[file_path]
-                        return new_provers
-                entry: Optional[Tuple[str, Set[str]]] = self.plot_filename_paths.get(file_path.name)
-                if entry is not None:
-                    loaded_parent, duplicates = entry
-                    if str(file_path.parent) in duplicates:
-                        log.debug(f"Skip duplicated plot {str(file_path)}")
-                        return new_provers
+                return None
+            if not file_path.exists():
+                return None
```

Comment:
+1 for an early exit instead of an extra indentation level for the whole function